eth/contracts/libraries/LibPermissions.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.0;
+
+import {OwnableStorage} from "@solidstate/contracts/access/ownable/OwnableStorage.sol";
+
+library LibPermissions {
+    using OwnableStorage for OwnableStorage.Layout;
+
+    event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
+
+    function contractOwner() internal view returns (address contractOwner_) {
+        contractOwner_ = OwnableStorage.layout().owner;
+    }
+
+    function enforceIsContractOwner() internal view {
+        require(msg.sender == OwnableStorage.layout().owner, "LibDiamond: Must be contract owner");
+    }
+
+    function setContractOwner(address _newOwner) internal {
+        address previousOwner = OwnableStorage.layout().owner;
+        OwnableStorage.layout().owner = _newOwner;
+        emit OwnershipTransferred(previousOwner, _newOwner);
+    }
```

Comment:
This isn't used anymore, right? And then you can remove the OwnershipTransferred event too

Reply:
[fd20a97](https://github.com/dfdao/game/pull/7/commits/fd20a97b4b619b2c43cd399b5522f73b794917c9)